src/cdk-new/get-hash.ts

```diff
@@ -0,0 +1,9 @@
+import {createHash} from 'crypto';
+
+export function getHash(...values: string[]): string {
+  return createHash(`sha1`)
```

Comment:
not really important properly, but `sha1` should not be used anymore.

Reply:
Does not matter in non-cryptographic contexts. It is only a matter of collision avoidance here.